### DISCUSSION TITLE
.NET MAUI Toolkit Community Standup

### DISCUSSION THREAD

**Discussion by GUOLDEV** [CommunityToolkit/Maui#173] [Q&A] [answered]
When is the next .NET MAUI Toolkit Community Standup?

**Comment by TheCodeTraveler** [comment, marked as answer]
We live stream our monthly standup on the 1st Thursday of every month @ 1200 Pacific Time. 

You can find all of the previously recorded stand ups on the .NET Foundation YouTube channel: https://m.youtube.com/watch?v=3XSIcI466rA

**Reply by GUOLDEV** [reply]
Thanks @brminnick 👍

**Comment by CliffAgius** [comment]
Closed as answered
